.github/workflows/codeql-analysis.yaml

```diff
@@ -43,8 +43,10 @@ jobs:
 
     - name: "Build"
       run: |
-        cmake .
-        make
+        mkdir build
+        cd build
+        cmake .. -G Ninja
+        ninja
```

Comment:
Should we have matrix builds with ninja and make?  (They _should_ be the same, but who knows?)